repos/spack_repo/builtin/packages/geomodel/package.py

```diff
@@ -93,6 +105,9 @@ class Geomodel(CMakePackage):
         depends_on("coin3d")
         depends_on("soqt")
         depends_on("gl")
+        depends_on("egl", when="@6.11:")
+
+    depends_on("googletest", when="@6.11:")
```

Comment:
I think this is then ONLY needed on 6.11, right? Starting from 6.12 the tests are not built, so GTest is not needed.

Reply:
Resolved by making it a build dependency in 6.11 and a test dependency in 6.12:.